src/System.Data.OleDb/tests/OleDbParameterTests.cs

```diff
@@ -17,9 +17,9 @@ public void OleDbParameterCollection_MultipleScenarios_Success()
             OleDbParameterCollection opc = command.Parameters;
 
             Assert.True(opc.Count == 0);
-            Assert.False(((Collections.IList)opc).IsReadOnly);
-            Assert.False(((Collections.IList)opc).IsFixedSize);
-            Assert.False(((Collections.IList)opc).IsSynchronized);
+            Assert.False(opc.IsReadOnly);
```

Comment:
I guess this shows that overriding the APIs is pointless. But it seems to exist in .NET Framework and in other providers (though SqlClient is missing IsSynchronized)